language/Turkish/strings.po

```diff
@@ -2086,7 +2090,7 @@ msgstr ""
 
 #: src/frontend/mame/ui/miscmenu.cpp:630
 msgid "Name:             Description:\n"
-msgstr ""
+msgstr "Ad:             Açıklama:\n"
```

Comment:
This is used as a heading in a list file where the first column is padded to a fixed width with spaces.  Could you please add spaces so the column width matches in the translation?  It needs two more spaces.

Reply:
added two more spaces